lasso/roadway.py

```diff
@@ -1753,7 +1753,7 @@ def write_roadway_as_shp(
 
         # unless specified that all the data goes to the DBF, only output A and B
         dbf_link_output_variables = (
-            dbf_link_output_variables if data_to_dbf else ["A", "B", "geometry"]
+            link_output_variables if data_to_dbf else ["A", "B", "geometry"]
```

Comment:
Apologies! I did one more test to write more output to the shape file for Rachel and found this bug. Please confirm this fix is correct and I'll merge into `develop`. Thanks.

Reply:
Your fix is right.  Thanks for finding it.